DependencyInjection/HttplugExtension.php

```diff
@@ -205,4 +219,25 @@ private function configureAuthentication(ContainerBuilder $container, array $con
                 ->addArgument(new Reference($authServiceKey));
         }
     }
+
+    /**
+     * Resolve the plugin enabled status if it is 'auto'.
+     *
+     * Returns false if plugin has no auto status allowed.
+     *
+     * @param string $name
+     * @param array  $pluginConfig
+     *
+     * @return bool
+     */
+    private function resolvePluginStatus($name, array $pluginConfig)
```

Comment:
then lets move the whole "auto" logic out of this PR. we can create a PR for the 2.0 version with it. and probably simply not merge it yet, as there is no hurry.